desko.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Desko page</title>
+    <script src="index.js"></script>
+</head>
+
+<body>
+    <h1>Desko</h1>
+</body>
+
+</html>
```

Comment:
There should be a `/n` here, but not an issue now. Just, for future reference.

Reply:
Should I merge to develop?

Reply:
Sure 👍